apps/web/src/pages/DirectivePage/utils.ts

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { IntlShape } from "react-intl";
+
+import { CardFlatProps, ExternalLinkProps } from "@gc-digital-talent/ui";
+
+interface GetFormLinkArgs {
+  formName: React.ReactNode;
+  files: { en: string; fr: string };
+  intl: IntlShape;
+}
+
+export const getFormLinks = ({
```

Comment:
Could we have some comments for these util functions?

Reply:
Added some documentation in [document directive link utils](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6143/commits/dbef551d6287e7d4e9f4b7a08e462d6884d097eb)

Let me know if it doesn't make sense or needs more.

Reply:
Perfecto!